packages/paste-website/src/pages/patterns/status/index.mdx

```diff
@@ -400,7 +400,7 @@ to ensure that the icon and the text are vertically aligned.
 
 ### Status badges
 
-For convience and consistency, we have created a `StatusBadge` (coming soon) component that can be used to compose a status icon with a text label.
+For convience and consistency, we have created a [`StatusBadge`](/components/status-badge) component that can be used to compose a status icon with a text label.
```

Comment:
```suggestion
Use the [`StatusBadge`](/components/status-badge) component to compose a status icon with a text label, usually when a status is shown outside of a Table. Common placements are near avatars (particularly for connectivity statuses), in cards, and in lists.
```

Reply:
todo: remove the "Other common placements" section at the bottom of this page, and move the example from that section up here.